### ISSUE TITLE
No download all tweet video

### ISSUE THREAD

**Issue by marciglesias17** [mmpx12/twitter-media-downloader#20]
I am running the command to download retweets from an account, which are video only, with the -U option to only download the ones that are not downloaded and with the -L option to log in with my account and not download all the videos. A retweet from a few days ago is not downloaded, but others are.

In the result only appears "XXXX.mp4: already exists" I don't see any message other than that (apparently all downloaded).

**Command:**
`twmd -u ACCOUNT -o /home/test/ -v -U -r -L`

**Comment by marciglesias17** [comment]
I add information: 

I look for one of the tweets that does not download en masse and I download it manually, indicating the tweet ID and it downloads correctly.

**Comment by marciglesias17** [comment]
Also, when I set the flag "-a" and "-U" and run the command several times to update the latest videos, some videos are duplicated.

**Comment by mmpx12** [comment]
@marciglesias17 send link for testing please. For dupplicated videos, do they have the same name and same checksum?

**Comment by marciglesias17** [comment]
I have found this problem in several examples, I give you an example with my twitter profile @marciglesias17

`twmd -u marciglesias17 -o /opt/media/ -a -U -r -L -B`

All videos have been downloaded except this one https://twitter.com/Alvama_Ice/status/1318874208437415936

In this example, I get the duplicate videos. If it has the same checksum and name, **RE-** is added to the beginning of the name.

Example:
4wa3htalRzgrDHfF.mp4
RE-4wa3htalRzgrDHfF.mp4

**Comment by marciglesias17** [comment]
The videos and photos in this tweet have also not been downloaded https://twitter.com/Merocle/status/1603064464034734082

**Comment by marciglesias17** [comment]
I attach the output of the command, the folder was completely empty, executed only once.

```
Downloaded rI99BtxjVY8gUGnt.mp4
Downloaded rI99BtxjVY8gUGnt.mp4
https://pbs.twimg.com/media/Fj2R3NOWQAE3tkc.jpg
Downloaded Fj2R3NOWQAE3tkc.jpg
Downloaded rI99BtxjVY8gUGnt.mp4
https://pbs.twimg.com/media/CsaXFFjWYAAmeYz.jpg
https://pbs.twimg.com/media/EoV6oJdXEAAsxRZ.jpg
https://pbs.twimg.com/media/EoV6oqkXYAINejI.jpg
https://pbs.twimg.com/media/CsbHZO_WgAA4MQD.jpg
https://pbs.twimg.com/media/C8QMRVoWsAAX8W6.jpg
Downloaded CsaXFFjWYAAmeYz.jpg
Downloaded CsbHZO_WgAA4MQD.jpg
Downloaded EoV6oJdXEAAsxRZ.jpg
Downloaded EoV6oqkXYAINejI.jpg
eaAuc1zSF6vY_Gbn.mp4: already exists
Downloaded C8QMRVoWsAAX8W6.jpg
8-_fQGia_gS-ztAO.mp4: already exists
Downloaded eaAuc1zSF6vY_Gbn.mp4
Downloaded eaAuc1zSF6vY_Gbn.mp4
4wa3htalRzgrDHfF.mp4: already exists
t_zLph4D6HFff7Ak.mp4: already exists
Downloaded 4wa3htalRzgrDHfF.mp4
Downloaded 4wa3htalRzgrDHfF.mp4
Downloaded t_zLph4D6HFff7Ak.mp4
Downloaded t_zLph4D6HFff7Ak.mp4
Downloaded 8-_fQGia_gS-ztAO.mp4
Downloaded 8-_fQGia_gS-ztAO.mp4
Downloaded GrO2y_E4sXf6bNXn.mp4
Downloaded GrO2y_E4sXf6bNXn.mp4
Downloaded GrO2y_E4sXf6bNXn.mp4

```

**Comment by mmpx12** [comment]
@marciglesias17  For the duplicate i think its because he retweet his tweet so 4wa3htalRzgrDHfF.mp4 is the original post and RE-4wa3htalRzgrDHfF.mp4 is a retweet. For tweet that aren't downloaded i think i use to much thread(goroutine). Will check this. I'll add a flag for debug and a flag for setting the number of goroutine (maybe with less it will work).

**Comment by marciglesias17** [comment]
@mmpx12 Do you foresee when you will have an update?

**Comment by mmpx12** [comment]
@marciglesias17  Twmd is based on https://github.com/n0madic/twitter-scraper ... Maybe open an issue in twitter-scrapper repo... Unfortunately there is nothing i can do aboit that.

**Comment by marciglesias17** [comment]
@mmpx12 I have opened a thread in the repository you mentioned, they recommend a change to fix this, I don't know what I should answer as I don't understand how this is programmed (I am not a programmer).

**Comment by mmpx12** [comment]
Have you tried to use "-L|--login" ?

**Comment by marciglesias17** [comment]
Yes, my command used is: **twmd -u ACCOUNT -o /home/test/ -v -U -r -L**

**Comment by marciglesias17** [comment]
Can you think of a solution?
